_sass/_docs.scss

```diff
@@ -1,3 +1,92 @@
+
+.sidebar {
+    top: 70px;
+    left:0;
+    width: $sidebar-width; 
+    height: 100vh;
+    overflow-y: auto;
+    overflow-x: hidden;
+    flex-grow: 2;
+    padding: 30px 30px 0px 10px;
+    position: sticky;
+    background-color: #e6e4e4;
+    text-align: right;
+    @media (max-width: $desktop-width) {
+        display: none;
+    }
+}
```

Comment:
Looking at how you styled the sidebar, this would explain the viewport clipping bug I mention at the start of this thread. Instead of using `overflow-x: hidden`, `padding: npx mpx;`, and properties of that nature, what you tried using [flexbox](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Flexible_Box_Layout/Basic_Concepts_of_Flexbox)?

Also, rather than `height: 100vh`, we could try using `height: 100%`.